### DISCUSSION TITLE
do we have the support for the todo item in the markdown editor

### DISCUSSION THREAD

**Discussion by Kellermaan** [requarks/wiki#7392] [Help / Questions] [answered]
do we have the support for the todo item in the markdown editor.
something like 
- [ ]

**Comment by fnafox** [comment, marked as answer]
https://docs.requarks.io/editors/markdown#task-lists

```md
- [x] Checked task item
- [x] Another checked task item
- [ ] Unchecked task item
```

**Reply by Kellermaan** [reply]
Oh, I see. sorry I missed the content in the end of checkbox.
if you use
 - [x]
 it can't be renderd to the checkbox.

Thanks a lot!

**Reply by Kellermaan** [reply]
is it right in the visual editor.
the checkbox looks like 
![image](https://github.com/user-attachments/assets/a812c532-9363-4487-804c-4ce4ab56816a)
